src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -168,12 +151,17 @@ def update_stats(self, area, simulation_status):
             self.file_export_endpoints.traded_energy_profile_redis)
 
     def _update_tree_summary(self, area):
-        price_energy_list = export_price_energy_day(area)
+        price_energy_list = self.price_energy_day.csv_output
 
         def calculate_prices(key, functor):
             # Need to convert to euro cents to avoid having to change the backend
             # TODO: Both this and the frontend have to remove the recalculation
```

Comment:
Should be use this opportunity to communicate with the frontend to remove the rounding and euro-converting on their side (if not already happend) and remove this comment?

Reply:
I agree, and I also suggest to use Euro cents for that. Right now the only displayed values will be trade rates, which make more sense to me in cents/kWh rather than Euro/kWh. I would suggest to also remove conversion on our side too and remain consistent to the other graphs. What do you think ?

Reply:
Perfect, thanks!